glutin/src/api/egl/display.rs

```diff
@@ -231,6 +231,91 @@ impl Display {
         }
     }
 
+    /// Create a sync.
+    ///
+    /// Creating a sync fence requires that an EGL context is currently bound.
+    /// Otherwise [`ErrorKind::BadMatch`] is returned.
+    ///
+    /// This function returns [`Err`] if the EGL version is not at least 1.5
+    /// or `EGL_KHR_fence_sync` is not available. An error is also returned if
+    /// native fences are not supported.
+    pub fn create_sync(&self, native: bool) -> Result<super::sync::Sync> {
+        if self.inner.version < super::VERSION_1_5
```

Comment:
```suggestion
        if self.inner.version < VERSION_1_5
```

Do a proper `crate` style import.